modules/ROOT/pages/dataweave-memory-management.adoc

```diff
@@ -4,20 +4,20 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: studio, anypoint, esb, transform, transformer, format, aggregate, rename, split, filter convert, xml, json, csv, pojo, java object, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping, streaming, weaveoutputhandler
 
-When processing large files through DataWeave in Mule, there are a few things you can set up to fine-tune how much memory will be used up and when.
-//LINK TO DW 1.0 LANDING PAGE IN MULE 3.9 DOCS:
-include::partial$dataweave1-links.adoc[tag=dataweave1LandingPage]
+When processing large files through DataWeave in Mule runtime engine, there are a few things you can set up to fine-tune how much memory will be used and when.
 
 [[buffersize]]
-== Memory vs Disk Usage
+== RAM vs Disk Usage
 
-DataWeave uses the system's memory as a buffer while processing a transformation unless a certain threshold is exceeded, in which case it resorts to using the system's hard disk as a buffer. By default, this threshold is set at *1572864 bytes*, but this value may be changed. The value refers to memory usage of each individual Transform component, not to an aggregate of all the ones in the project.
+DataWeave uses disk for storing information and not running out of memory. The files that are going to be created are going to be placed on the temporary directory. If you desire a custom directory instead of the system default, you can specify the directory using the `java.io.tmpdir` property.
 
-To change the threshold value at which memory is no longer used as a buffer, you must add a system property `com.mulesoft.dw.buffersize` and assign it the number (in bytes) of your new threshold.  System properties may be defined in several ways. See xref:configuring-properties.adoc#system-properties[system properties] for more details and more ways you can set these.
-
-The value you assign to this property affects your entire Mule application, affecting each instance of the Transform component individually.
+All DataWeave files are going to have the prefix `dw-buffer-` and there are two kind of files that are being generated. +
 
+- `dw-buffer-output-${count}.tmp` is used to store the output of a transformation when the result is bigger than the threadshold *1572864 bytes*.
+To change the threshold value at which memory is no longer used as a buffer, you must add the system property `com.mulesoft.dw.buffersize` and assign it the number (in bytes) of your new threshold.  System properties may be defined in several ways. See xref:configuring-properties.adoc#system-properties[system properties] for more details on how to set them.
```

Comment:
```suggestion
To change this threshold value, add the system property `com.mulesoft.dw.buffersize` and assign it the number of bytes you want as your new threshold. Because you can define system properties in several ways, see xref:configuring-properties.adoc#system-properties[system properties] for further details.
```